url-updater/src/main/java/com/devonfw/tools/ide/url/updater/node/NodeUrlUpdater.java

```diff
@@ -28,13 +34,19 @@ protected String getGithubRepository() {
   @Override
   protected void addVersion(UrlVersion urlVersion) {
 
-    String baseUrl = "https://nodejs.org/dist/${version}/node-${version}-";
-    doAddVersion(urlVersion, baseUrl + "win-x64.zip", WINDOWS);
-    doAddVersion(urlVersion, baseUrl + "win-aarch64.zip", WINDOWS, ARM64, "");
-    doAddVersion(urlVersion, baseUrl + "linux-x64.tar.gz", LINUX);
-    doAddVersion(urlVersion, baseUrl + "linux-aarch64.tar.gz", LINUX, ARM64, "");
-    doAddVersion(urlVersion, baseUrl + "darwin-x64.tar.gz", MAC);
-    doAddVersion(urlVersion, baseUrl + "darwin-aarch64.tar.gz", MAC, ARM64, "");
+    String version = urlVersion.getName();
+
+    String baseUrl = "https://nodejs.org/dist/v${version}/node-v${version}-";
```

Comment:
you are breaking compatibility with this change:
https://github.com/devonfw/ide-mirrors/blob/master/node/urls

I already had discussions with @CREITZ25 on these topics and I said that typically a version number should not contain static artificial prefixes or infixes. There are even cases (for other tools) where in the URL the version is once with such prefix and another time without it. As adding such prefix like `v` via `v${version}` is easy while removing it is not, we have cases where we had already been forced to remove it. However, there has even been a case where for the same tool versions with such prefix as well as others without it existed so it always becomes tricky because of the mess that different tool vendors create and the lack of a commonly accepted best-practice.

I am OK with changing this but then we have to add it to the migration (#1087).

Reply:
I use `mapVersion` function to add `v` to maintain compatibility